src/main/java/org/junitpioneer/vintage/ExpectedExceptionExtension.java

```diff
@@ -11,7 +11,7 @@
 package org.junitpioneer.vintage;
 
 import static java.lang.String.format;
-import static org.junit.platform.commons.util.AnnotationUtils.findAnnotation;
+import static org.junit.platform.commons.support.AnnotationSupport.findAnnotation;
```

Comment:
No it's not possible at the moment, because the `PioneerAnnotationUtils` are package private and both extensions are in the vintage package. If they weren't I guess we would have already changed them, when we introduced our helper class.